### PR TITLE
Cloud documentation reorganization

### PR DESCRIPTION
**Pull request by justinegeffen**
Restructuring documentation to add Cloud docs and reorganize admin guide.

Jira: https://mattermost.atlassian.net/browse/MM-29596

### CODE REVIEW

**Comment by justinegeffen** [comment]
@amyblais, this content has all been copied over from our existing content. Minor changes to the content have been made, such as removing reference to "versions prior to 5.12" and some formatting fixes (which I submitted in PRs for this repo. So it's not 45+ files of new content. :)

**Review by adamjclarkson — APPROVED** [review]

**Comment by amyblais** [comment]
Some of the pages like cloud billing, cloud reporting, and cloud compliance look a bit empty, but I assume these will be filled in the future? Just want to confirm before approving the PR.

**Comment by wiersgallak** [comment]
@justinegeffen Let's copy in this page for reporting: https://mattermost.github.io/docs/justinegeffen/administration/statistics.html 

@amyblais The PR for the billing page is here: https://github.com/mattermost/docs/pull/4056.  Compliance is filled out for me.

**Comment by justinegeffen** [comment]
> Some of the pages like cloud billing, cloud reporting, and cloud compliance look a bit empty, but I assume these will be filled in the future? Just want to confirm before approving the PR.

We're going to be moving that today so I'll let you know when that's done. :)

File: source/cloud/cloud-administration/cloud-compliance.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Daily compliance reports may also be generated, supporting integration with compliance solutions like `Global Relay <https://docs.mattermost.com/administration/compliance.html#global-relay-support>`__. 
+
+By default, all Mattermost Editions retain all messages, including edits and deletes, along with all files uploaded. 
+
+Enabling Compliance Reporting 
+=============================
+
+After purchasing and installing a license key for Enterprise Edition E20: 
+
+1. Go to **System Console > Compliance > Compliance Monitoring** (or **System Console > General > Compliance > Enable Compliance** in versions prior to 5.12) and set the **Enable Compliance Reporting** value to **true**.
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/cloud-compliance.rst:25]:
```suggestion
1. Go to **System Console > Compliance > Compliance Monitoring** and set the **Enable Compliance Reporting** value to **true**.
```

File: source/cloud/cloud-administration/cloud-compliance.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+3. Click **Save**. 
+
+This will enable the option to generate Daily Compliance Reports.
+
+Turn on Daily Compliance Reports 
+================================
+
+After enabling compliance reporting: 
+
+1. Go to **System Console > Compliance > Compliance Monitoring** (or **System Console > General > Compliance > Enable Daily Report** in versions prior to 5.12) and set the **Enable Daily Report** value to **true**.
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/cloud-compliance.rst:36]:
```suggestion
1. Go to **System Console > Compliance > Compliance Monitoring** and set the **Enable Daily Report** value to **true**.
```

File: source/cloud/cloud-administration/cloud-compliance.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+2. Click **Save**. 
+
+Your system will now export all new messages posted within a 24-hour period as a ``.csv`` file to the location specified in **Compliance Report Directory**. This feature can be used in conjunction with centralized compliance reporting systems that move.
+
+Run Compliance Reports  
+======================
+
+Compliance Reports are exports of all messages in Mattermost that match the report criteria. To run a report: 
+
+1. Go to **System Console > > Compliance > Compliance Monitoring** (or **System Console > Compliance > Compliance Monitoring** in versions prior to 5.12).
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/cloud-compliance.rst:46]:
```suggestion
1. Go to **System Console > > Compliance > Compliance Monitoring**.
```

File: source/cloud/cloud-user-management/user-management.rst [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+---------------------
+
+If you have the required permissions, you can add people to your team via a direct invite, a public team invite link, or if they already have an account on the server, you can add them to the team yourself.
+
+From version 5.16, the System Admin can `enable guest access <https://docs.mattermost.com/deployment/guest-accounts.html>`_. A guest is a special role that is restricted to only the channels they have been invited to. When this is enabled, you can invite guests or users external to the organization such as vendors or customers.
+
+Direct Invites
+~~~~~~~~~~~~~~
+
+Direct invites are invitation emails sent from your team's server directly to the invited person's email address. A link within the invitation directs them to an account creation page. Invitation links sent by email expire after 48 hours and can only be used once. A System Admin can invalidate all active invitation links via **System Console > Authentication > Signup > Invalidate pending email invites** (or **System Console > Security > Sign Up > Invalidate pending email invites** in versions prior to 5.12).
```
Comment (justinegeffen) [inline source/cloud/cloud-user-management/user-management.rst:14]:
```suggestion
Direct invites are invitation emails sent from your team's server directly to the invited person's email address. A link within the invitation directs them to an account creation page. Invitation links sent by email expire after 48 hours and can only be used once. A System Admin can invalidate all active invitation links via **System Console > Authentication > Signup > Invalidate pending email invites**.
```

File: source/administration/devops-command-center.rst [hunk 5]
```diff
@@ -1,6 +1,6 @@
-======================
-DevOps Command Center
-======================
+==============================
+Mattermost Incident Management
+==============================
 
 Mattermost Incident Management is the first workflow that we officially support as part of DevOps Command Center.
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:5]:
```suggestion
```

Comment (justinegeffen) [reply source/administration/devops-command-center.rst:5]: @itao, I've removed the opening line as rewording it without DevOps Command Center fell flat. LMK your thoughts.

File: source/cloud/cloud-administration/cloud-compliance.rst [hunk 6]
```diff
@@ -0,0 +1,145 @@
+
+Compliance Reporting and Oversight (E20)
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/cloud-compliance.rst:2]:
```suggestion
Compliance Reporting and Oversight
```

File: source/cloud/cloud-administration/cloud-compliance.rst [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+----------------------------------------
+
+Available in `Enterprise Edition E20 <https://about.mattermost.com/pricing/>`__.
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/cloud-compliance.rst:5]:
```suggestion
```

File: source/cloud/cloud-administration/compliance-export.rst [hunk 8]
```diff
@@ -0,0 +1,111 @@
+
+Compliance Export Beta (E20)
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/compliance-export.rst:2]:
```suggestion
Compliance Export Beta
```

File: source/cloud/cloud-administration/compliance-export.rst [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+============================
+
+Available in `Enterprise Edition E20 <https://about.mattermost.com/pricing/>`__.
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/compliance-export.rst:5]:
```suggestion
```

**Comment by justinegeffen** [comment]
@amyblais, I've removed the empty compliance file and we've done some more editing (removing content, and adjusting headings). If you have the bandwidth for a review that'd be great. :) We plan to merge this asap and then continue working on the docs.

**Comment by amyblais** [comment]
I don't see content for ``/cloud/cloud-administration/create-new-workspace.rst``.
Also, a few pages are duplicated such as ``cloud-incoming-webhooks.rst`` and ``incoming-webhooks.rst``, and ``cloud/mobile-apps-faq.rst`` and ``cloud-mobile/mobile-apps-faq.rst``.

**Comment by wiersgallak** [comment]
@amyblais  I will edit post merge and ensure we have all the missing pages.

**Pull request merged**